waku-sys/Cargo.toml

```diff
@@ -19,4 +19,4 @@ crate-type = ["rlib"]
 [dependencies]
 
 [build-dependencies]
-bindgen = "0.64"
\ No newline at end of file
+bindgen = "0.64"
```

Comment:
wtf?

Reply:
My formater added newline symbol at the on of a file. It might be a problem if yours removes it.

Reply:
that is ok. But the diff just shows the same line without change 😅  so I got confused.